xen/include/xen/slr_table.h

```diff
@@ -0,0 +1,254 @@
+/*
+ *  SPDX-License-Identifier: GPL-3.0
```

Comment:
Xen coding style has requirements for [single-line SPDX header](https://github.com/TrenchBoot/xen/blob/master/CODING_STYLE#L20), but there are many files that don't follow it, or even don't have SPDX tags at all, so I'm not sure what to do about it. Perhaps it would be safest to move this to separate comment, but in any case we will have to re-check for proper licenses and copyrights before upstreaming, so we can postpone it until then and check all our changes at once.

Reply:
Might as well do it now: https://github.com/TrenchBoot/xen/compare/5ed262bc0df7696f824b5a547339a804fa9e5a52..39a1478db53029b133c75d4f15a0f5774ad92f8a#diff-609762c7fad93e82743d840bbbfd586771646f31761b1b8bd54bd4384a591dfbR1-R2